srcpkgs/jose/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'jose'
+pkgname=jose
+version=10
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --disable-openmp"
+hostmakedepends="pkg-config"
+makedepends="jansson-devel openssl-devel zlib-devel"
+short_desc="C implementation of Javascript Object Signing and Encryption"
+maintainer="Jiri Belka <jirib79@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/latchset/jose"
+distfiles="https://github.com/latchset/jose/releases/download/v${version}/jose-${version}.tar.bz2"
+checksum=5c9cdcfb535c4d9f781393d7530521c72b1dd81caa9934cab6dd752cc7efcd72
+
+post_install() {
+	rm -rf usr/lib64
+}
```

Comment:
Don't use `-f` unless necessary, otherwise it won't complain if the file doesn't exist (which would mean the command doesn't have to be called).

Given that `usr/lib64` (in the destdir, so I'm not sure you're actually hitting any existing directory here) is a symlink to `usr/lib` for 64-bit targets, which is removed as part of the packaging process, what is the purpose of this?